src/hearings/components/hearing-confirmation/hearing-confirmation.component.html

```diff
@@ -10,10 +10,13 @@ <h2 class="govuk-heading-m">{{subheading}}</h2>
         <div class="govuk-body" *ngIf="subheadingDescription">
           {{subheadingDescription}}
         </div>
-        <div class="govuk-body" *ngIf="caseId">
+        <div class="govuk-body" *ngIf="caseId && !hearingId">
           You can <a href="javascript:void(0)"
             [routerLink]="['/', 'cases', 'case-details', caseId, 'hearings']" class="govuk-link">view the status of this hearing in the hearings tab</a>.
         </div>
+        <div class="govuk-body" *ngIf="caseId && hearingId">
+          <a href="javascript:void(0)" [routerLink]="['/', 'cases', 'case-details', caseId, 'hearings']" class="govuk-link">Return to hearings</a>
+        </div>
```

Comment:
seems this is not necessary, could you ask Adeal whether we can use the same message 'view the status of this hearing in the hearings tab'

Reply:
I talked to Aleena, and she said that the wording does not match for linked hearings, and it should match the prototype.

Reply:
this wording it actually has the same/similar meaning, and the url is same as well, we should ask them to provide a generic message if we put this in a common component, otherwise we shouldn't put this into a common componnent for just two different kind of message. Or you could pass the message into the @Input. That hearingId in ngIf is redundant and useless.

Reply:
Last time we tried and decided not to use @input as the wording also contains hyperlink.
I think generic message "Return to hearings" should cater for all scenarios.

Reply:
Thank you @rayliangatos I have now made the changes.